app.py

```diff
@@ -149,6 +153,35 @@ def get_book_info():
     )
 
 
+@app.route("/get_favorites", method="POST")
+def get_favorites():
+    favorites = Favorites.query.filter(Favorites.userEmail == current_user).all()
+    favorite_list = []
+    for i in favorites:
+        favorite_list.append(i.isbn)
+
+    # Need to make api calls for every isbn in the list and get new list based on that info and then pass that into html.
+
+
+# Need current user to store current user, so need to talk to maryam about user login
+# And make sure that what i am doing will work.
+@app.route("/add_favorite", method=["GET", "POST"])
```

Comment:
Keep this route. This will work after we add a form to add the ISBN of a book as a new favorite.